pkgs/stdenv/darwin/make-bootstrap-tools.nix

```diff
@@ -100,6 +100,8 @@ in rec {
         cp ${cctools_}/bin/$i $out/bin
       done
 
+      cp -d ${darwin.libtapi}/lib/libtapi* $out/lib
```

Comment:
Is this actually needed? If so, we will have to regenerate bootstrap tools

Reply:
`libtapi` is enabled everywhere, so the `ld` that's included in the bootstrap tools now depends on `libtapi`. The intent is to make sure that the bootstrap tools can be updated, but this change doesn't require a bootstrap tools update.

Removing this causes
```
$ nix-build pkgs/stdenv/darwin/make-bootstrap-tools.nix -A test
[...]
dyld: Library not loaded: @rpath/libtapi.dylib
  Referenced from: /nix/store/ym4nscqv1h7sxn6b6dby0i4106lcw9sr-unpack/bin/ld
  Reason: image not found
```

I do want to regenerate the bootstrap tools as part of my overall goals. Next step is a `.tbd` based libSystem which I have on a branch. It can link `hello` on Big Sur, but there's still a decent amount of work before I can open that as a PR.